### ISSUE TITLE
Can't see URDFModel 

### ISSUE THREAD

**Issue by guhur** [RoboStack/jupyter-ros#21]
Hi,

Thanks a lot for this great project! 

Unfortunately, I can't play correctly the notebooks.

Here is a screenshot of `ROS robo` to compare to a screenshot from `rviz`.

![Screenshot from 2019-06-28 10-24-08](https://user-images.githubusercontent.com/12297742/60328749-36300300-998f-11e9-8fc1-a1378edc49d4.png)
![Screenshot from 2019-06-28 10-24-20](https://user-images.githubusercontent.com/12297742/60328750-36300300-998f-11e9-9680-964abc9b4b0d.png)

I have no interesting message on the `roslaunch` outputs.

Thanks!

**Comment by wolfv** [comment]
Yes, getting the robot to show up is (unfortunately) not yet super straight forward.

Did you launch all launch files as indicated (especially the bridge.launch?)

Did you convert the TIFF textures to PNG as mentioned?
Did you start a server at localhost:3000? (But I am pretty sure it fails before that, otherwise you would see some blocks popping up with no texture).
